catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/CachingFederationStrategy.java

```diff
@@ -536,27 +525,53 @@ public void run() {
 
         public CallableSourceResponse(Source source, QueryRequest request) {
             this.source = source;
-            this.request = request;
+            this.request = new QueryRequestImpl(request.getQuery(), new HashMap<>(request.getProperties()));
         }
 
         @Override
         public SourceResponse call() throws Exception {
-            final SourceResponse sourceResponse =
-                    source.query(new QueryRequestImpl(request.getQuery(), request.getProperties()));
+            SourceResponse sourceResponse = null;
+            try {
+                sourceResponse = source.query(new QueryRequestImpl(request.getQuery(),
+                        request.getProperties()));
+            } catch (UnsupportedQueryException e) {
+                LOGGER.info("Error querying source {}", source.getId(), e);
+            }
+
+            QueryResponse queryResponse = new QueryResponseImpl(request,
+                    sourceResponse == null ? new ArrayList<>() : sourceResponse.getResults(),
+                    true,
+                    sourceResponse == null ? 0L : sourceResponse.getHits(),
+                    sourceResponse == null ? new HashMap<>() : sourceResponse.getProperties());
+
+            for (PostFederatedQueryPlugin service : postQuery) {
+                try {
+                    queryResponse = service.process(queryResponse);
+                } catch (PluginExecutionException | StopProcessingException e) {
+                    LOGGER.info("Error executing PostFederatedQueryPlugin", e);
```

Comment:
I think we want a StopProcessingException to stop further plugins from getting processed.